### PR TITLE
Added social profiles. Update footer.html

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes issue#109
Updated footer.html

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
https://arjunpandey2003.github.io/2012.fossasiarjunp.org/

**Comment by arjunpandey2003** [comment]
why is my vk logo coming out like that

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Squash! :+1:

**Comment by arjunpandey2003** [comment]
@0x48piraj can u tell me why my vk logo is coming out like that

**Comment by 0x48piraj** [comment]
image

**Review by isuruAb — COMMENTED** [review]
some icons are missing. please check

**Review by anshumanv — COMMENTED** [review]
Test link not working.

**Comment by arjunpandey2003** [comment]
@anshumanv https://arjunpandey2003.github.io/2012.fossasiarjunpcde.org/.

File: _includes/footer.html [hunk 1]
```diff
@@ -37,8 +37,9 @@
                                 <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
-                                <li><a href="https://plus.google.com/108920596016838318216" target="default"><i class="icon social_googleplus"></i></a></li>
-                                <li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
+                                <li><a href="https://plus.google.com/108920596016838318216" target="default"><i class="icon social_googleplus"></i></a></li>         
+                                <li><a target="_self" href="https://blog.fossasia.org/"><i class="fa fa-rss"></i></a></li>          
+                                <li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
```
Comment (anshumanv) [inline _includes/footer.html]: remove `fa-lg` from the class

Comment (anshumanv) [reply _includes/footer.html]: See other's comments too.

**Pull request closed without merging**